### ISSUE TITLE
Error in running tests in TQ3lib-clean

### ISSUE THREAD

**Issue by peymanDavvaloKhongar** [sundmanbo/opencalphad#39]
Testing F90 test cases in TQ3lib-clean, it seems that many of them are written for the older versions of Open Calphad and they do not work with versions 4 and 5. An example will be compiling 'liboct.F90' in test1 that will raise error  
```
liboctq.F90:638:46:

           call get_phase_name(phcs(n1)%phaseix,phcs(n1)%compset,name)
                                                                   1
Error: ‘phaseix’ at (1) is not a member of the ‘gtp_phasetuple’ structure
```
Here is the declaration:
```
  type(gtp_phasetuple), dimension(maxp) :: phcs
```
looking at the type 'gtp_phasetuple' in models/gtp3.F90, does not have any member 'phaseix'. There is a comment to this type declaration in that file which refers to old code.

```
! >>>>>>>>>>>> old     integer phaseix,compset,ixphase,lokvares,nextcs
```

I recommend either to remove these test cases (or a note in a readme file stating these will not work with new version) or these test cases be corrected with the new version.

**Comment by sundmanbo** [comment]
I have not modified the TQ routines for more than a year, sorry for that.  I have now checked the crfe example in the F90 directory and it should work.  There were several problems due to the fact I added some C routines for the interactive version but they are now replaced by dummy routines.  There were also several changes in the data structure that had not been updated.  
For the Cpp examples I cannot do anything as I do not know C++
